sunpy/net/dataretriever/client.py

```diff
@@ -1,63 +1,45 @@
-import copy
 from pathlib import Path
-from collections import OrderedDict, namedtuple
-
-import numpy as np
+from collections import OrderedDict
 
 import astropy.table
 import astropy.units as u
+from astropy.time import TimeDelta
 
 import sunpy
 from sunpy import config
 from sunpy.net._attrs import Time, Wavelength
-from sunpy.net.attr import Range
 from sunpy.net.base_client import BaseClient, BaseQueryResponse
-from sunpy.time import TimeRange
+from sunpy.time import TimeRange, parse_time
 from sunpy.util.parfive_helpers import Downloader
+from sunpy.util.scraper import Scraper
 
 TIME_FORMAT = config.get("general", "time_format")
 
 __all__ = ['QueryResponse', 'GenericClient']
 
 
-class QueryResponseBlock:
+class QueryResponseBlock(OrderedDict):
```

Comment:
This PR feels like a good opportunity to get rid of this class? Or shall we do that later?

Reply:
I thought this was removed?

Reply:
I had to bring back `QueryResponseBlock` due to two reasons:
- All vso, jsoc, dataretriever individual records (rows in table) used to be classes, and to access their attributes, we have to do something like `row.url` instead of `row['ur']` in my implementation. So this ensured uniformity.
- A huge changes might have to be done in `database` tests, first by checking what type of response it is and whether to do `['url']` or `.url`. So it saved me from it.

Reply:
The question is, is it worth removing this class and just doing those annoying little changes everywhere?

Reply:
Removed in the latest commit, there weren't lot of changes.
Now the question is, should we keep the way to access fields from row data of QueryResponse (earlier QueryResponseBlock ) different for VSO and DR?
I think it would be rarely used. And after `:func:show` added, it won't be required.

Reply:
Different how?

Reply:
Like in VSO, the zeep client makes each block an object, so time is accessed like `block.time` or `getattr(block, 'time')`.

But I make them a dictionary, so it is accessed like `block['time']` or `block.get('time')`

Reply:
On the surface I would think we go for the latter, is it not how astropy.tables are indexed?

Reply:
Exactly, that's how we index cols in astropy table, like I'm doing in dict